apps/library/admin/events.py

```diff
@@ -15,12 +16,20 @@
 
 class ImagesInBlockInline(InlineReadOnlyMixin, admin.TabularInline):
     model = Performance.images_in_block.through
+    readonly_fields = ("preview",)
     verbose_name = "Изображение в блоке изображений"
     verbose_name_plural = "Изображения в блоке изображений"
     extra = 0
     max_num = 8
     classes = ["collapse"]
 
+    def preview(self, obj):
+        url = obj.image.image.url
+        return format_html(f'<img src="{url}" width="200" height="100" style="object-fit: contain;" />')
+
+    preview.short_description = "Превью"
+    Performance.images_in_block.through.__str__ = lambda self: ""
```

Comment:
```suggestion
    model.__str__ = lambda self: ""
```
или
```suggestion
    self.model.__str__ = lambda self: ""
```

Тогда вообще легко в миксин запихнуть будет